action.yml

```diff
@@ -100,6 +99,7 @@ runs:
     - id: check
       env:
         PLAN_ARGS: ${{ inputs.plan_args }}
+        TENV_AUTO_INSTALL: true
```

Comment:
Hmm, does your workflow define a terraform version? Or just relying on "latest"?

I though this line would handle things:

https://github.com/HENNGE/terraform-check/blob/d53c15796c0e65352aa0432db8dd883a26d3585c/action.yml#L129

but I did not test a workflow with no terraform version specified.

Reply:
@aclemons 
We do define a default version but generally rely on the .terraform-version file and we have that version auto-installed by terraform without explicitly setting it